.travis.yml

```diff
@@ -34,7 +34,6 @@ jobs:
          - cargo install cargo-kcov
          - kcov --version
```

Comment:
What about these two lines?

Reply:
They should go as well as the kcov installation above.